core/src/main/scala/org/apache/spark/api/python/PythonRunner.scala

```diff
@@ -65,6 +66,15 @@ private[spark] object PythonEvalType {
   }
 }
 
+object BasePythonRunner {
```

Comment:
```suggestion
private[spark] object BasePythonRunner {
```
should this be private?